### ISSUE TITLE
Expo crash after auth redirect to app via deep link

### ISSUE THREAD

**Issue by drfenixion** [expo/expo#4394]
Expo crash after auth redirect to the app via a deep link
Before it worked fine. I try to test my before commits for catch crash trigger but can't get it (in all worked commit I get crash). I think it can reference with expo app version (version updates time to time).

**How to reproduce:**
Any web auth script with a redirect to the app from a web browser via a deep link. (Android Expo)

**I did:**
- remove npm cache
- reinstall npm packages
- reinstall expo app
- test on another device

**In android adb logcat i see:**
```
06-03 23:04:45.202  6960  7057 E ExponentKernelModule: java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.String.length()' on a null object reference
06-03 23:04:45.401  6960  6960 W **unknown:WrappingUtils: Don't know how to round that drawable: com.facebook.drawee.drawable.RoundedColorDrawable@a43726e**
06-03 23:04:45.476  6960  6960 E **unknown:NativeViewHierarchyManager: Unable to update properties for view tag 199**
06-03 23:04:45.476  6960  6960 E unknown:NativeViewHierarchyManager: com.facebook.react.uimanager.IllegalViewOperationException: ViewManager for tag 199 could not be found
```

**Using - react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz"**

Any help plz. This crash block all my job.

**Comment by sjchmiela** [comment]
Could you please try opening _Expo APIs_ app on the offending device with the latest Expo client?

1. Close Expo Client.
2. Open Expo Client.
3. Go to _Explore_ tab (bottom tab bar).
4. Tap _Expo APIs_ row. (A new experience should open. Its source code is [here](https://github.com/expo/expo/tree/master/apps/native-component-list).)
5. There are multiple showcase screens. We will be interested in checking out `AppAuth`, `AuthSession`, `Google` and `GoogleSignIn`. Please go through them and check if you are able to reproduce the crash in any one of them.
6. If you can't, could you try creating a Snack that would reproduce the crash?

I tried to reproduce the crash on all the 4 screens using:
- device with Android 7.1,
- emulators with Android 8.0 and 7.0

and none crashed. 😢

**Comment by drfenixion** [comment]
I reinstall Expo from Play Market and restart the device. Test on a real device with Android 8.0.0.

Expo APIs get crashed in AppAuth, Google, AuthSession (second try I get a message like "Looks like something went wrong!" from https://login.auth0.com).

Fine auth process (without a crash) in GoogleSignIn, FacebookLogin (web, browser). But I think it is another type of auth. Because of GoogleSignIn authed without browser. FacebookLogin (web, browser) open auth form in a popup (not full browser window).

**Comment by cruzach** [comment]
Update- this should be fixed with the new Expo client for Android v2.11.2, which has now been published to Google Play 🎉
